### PR TITLE
Download modal

### PR DESCRIPTION
**Pull request by dlangston**

### CODE REVIEW

**Comment by dlangston** [comment]
@DanielJMaher

**Comment by dlangston** [comment]
screen shot 2015-08-12 at 11 45 03 am

**Comment by dlangston** [comment]
Old

screen shot 2015-08-12 at 11 46 45 am

**Comment by DanielJMaher** [comment]
@birdage

File: Gruntfile.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ module.exports = function(grunt) {
             "ooiui/static/js/partials/Banner.html",              
             "ooiui/static/js/partials/Navbar.html",
             "ooiui/static/js/partials/MenuToggle.html",
-            "ooiui/static/js/partials/LoggedInNavItems.html"
+            "ooiui/static/js/partials/LoggedInNavItems.html",
```
Comment (birdage) [inline Gruntfile.js]: is there is nothing after this line there should be no `,`

File: Gruntfile.js [hunk 2]
```diff
@@ -361,7 +361,7 @@ module.exports = function(grunt) {
             "ooiui/static/lib/metis-menu/dist/metisMenu.css",
             "ooiui/static/css/common/loginDemo.css",
             "ooiui/static/css/common/newEvent.css",
-            "ooiui/static/css/common/ModalForm.css"
+            "ooiui/static/css/common/ModalForm.css",
```
Comment (birdage) [inline Gruntfile.js]: @dlangston is there is nothing after this line there should be no `,`

File: Gruntfile.js [hunk 3]
```diff
@@ -1263,6 +1263,7 @@ module.exports = function(grunt) {
             "ooiui/static/css/common/newEvent.css",
             "ooiui/static/css/common/ModalForm.css",
             "ooiui/static/css/common/timeseries.css",
+            "ooiui/static/css/common/downloadModalForm.css",
```
Comment (birdage) [inline Gruntfile.js]: @dlangston is there is nothing after this line there should be no `,`

File: ooiui/static/js/views/science/StreamDownloadFormView.js [hunk 4]
```diff
@@ -83,11 +83,16 @@ var StreamDownloadFormView = Backbone.View.extend({
     var endDate = new Date(model.get('end'));
     this.$start_date_picker.setDate(startDate);
     this.$end_date_picker.setDate(endDate);
-    this.$el.find('.message h3').text(model.get('display_name'));
+    
+    if(model.get('display_name') != null) {
+      this.$el.find('#streamName').text(model.get('display_name'));
+    }else{
+    this.$el.find('#streamName').text(model.get('stream_name'));
```
Comment (birdage) [inline ooiui/static/js/views/science/StreamDownloadFormView.js]: @dlangston indent line 90 pls to match formatting

File: ooiui/static/js/views/science/StreamDownloadFormView.js [hunk 5]
```diff
@@ -83,11 +83,16 @@ var StreamDownloadFormView = Backbone.View.extend({
...
+    }
 
     this.$el.find('#type-select').val(selection);
 
-    this.$el.find('.stream-name').text(model.get('stream_name'));
+    // this.$el.find('.stream-name').text(model.get('stream_name'));
```
Comment (birdage) [inline ooiui/static/js/views/science/StreamDownloadFormView.js]: @dlangston remove comment line if not needed

**Comment by dlangston** [comment]
@DanielJMaher @birdage Made the changes, should be all set.

**Pull request merged**